src/Testing/InteractsWithMail.php

```diff
@@ -18,6 +20,40 @@ private function getMailer()
     {
         return $this->mailer ?: MailThief::getFacadeRoot();
     }
+    
+    /**
+     * @return MailThiefCollection
+     */
+    public function getMessages()
+    {
+        return $this->getMailer()->messages
+    }
+
+    /**
+     * @param array $emails
+     * @return MailThiefCollection
+     */
+    public function getMessagesFor(array $emails)
```

Comment:
If this intended to be used with singular and multiple emails being passed in; consider removing the typehint and add the following:

```php
if (! is_array($emails)) {
   $emails = [$emails];
}
```

This allows you to rewrite line 55 with `return $this->getMessagesFor($email)->last();`.